cms/static/js/certificates/models/certificate.js

```diff
@@ -13,6 +13,17 @@ function(Backbone, _, str, gettext) {
             version: 1
         },
 
+        relations: [{
```

Comment:
I think you also need to specify a 'collection' attribute here, based on what I've see at http://backbonerelational.org